### ISSUE TITLE
aggregation functions (

### ISSUE THREAD

**Issue by sebmarste** [Rostlab/JS18_ProjectA_Group1#62]

**Comment by syxolk** [comment]
How is it going on here?

**Comment by sebmarste** [comment]
i will start on that once column operations is finished, but im not sure whether i can du it before feature freeze...

**Comment by romikps** [comment]
are you planning on implementing this after #19 is done?

**Comment by sebmarste** [comment]
@romikps no, as feature freeze was last week, this is something that will most likely remain open :/
